docs/watch-options.md

```diff
@@ -203,7 +203,7 @@ Set `options: { cwd: { files: 'a/path', event: 'a/path' }}` to strip off `a/path
 
 
 ## options.livereloadOnError
-Type: `Boolean`  
-Default: `true`  
+Type: `Boolean`
```

Comment:
2 trailing spaces = line break in markdown.

The fact that it's missing in line 197 doesn't mean it should be removed here.